python/ray/autoscaler/_private/gcp/node_provider.py

```diff
@@ -164,14 +164,23 @@ def internal_ip(self, node_id: str):
             return ip
 
     @_retry
-    def create_node(self, base_config: dict, tags: dict, count: int) -> None:
+    def create_node(self, base_config: dict, tags: dict, count: int) -> Dict[str, None]:
+        """Creates instances.
+
+        Returns dict mapping instance id to None for the created
+        instances.
+        """
         with self.lock:
             labels = tags  # gcp uses "labels" instead of aws "tags"
 
             node_type = get_node_type(base_config)
             resource = self.resources[node_type]
 
-            resource.create_instances(base_config, labels, count)
+            results = resource.create_instances(base_config, labels, count)
+            instance_dict: Dict[str, None] = {}
+            for _, instance_id in results:
+                instance_dict[instance_id] = None
```

Comment:
eh...None feels a little weird.
Can we actually use the response dict (_) as the value?

Reply:
Good point. Have just updated that :)